lib/src/components/NxTableHead/NxTableHead.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import {addPropsToChildren} from '../../util/childUtil';
+
+import { Props } from './types';
+export {Props} from './types';
+
+const NxTableHead = function NxTableHead(props: Props) {
+  const {children, ...attrs} = props;
+
+  return (
+    <thead {...attrs}>
+      {addPropsToChildren(children, {isHeader: true})}
+    </thead>
+  );
+};
+
+export default NxTableHead;
```

Comment:
Missing propTypes declaration

Reply:
What would the propTypes even be? There aren't any special props for this component.

Reply:
> What would the propTypes even be? There aren't any special props for this component.

We not only list `propTypes` for special props; any prop that the component takes should be there.
See `lib/src/components/NxTextInput/types.ts` as an example.